lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.h

```diff
@@ -68,7 +68,7 @@ class MAYAUSD_CORE_PUBLIC UsdTRSUndoableCommandBase
     // UFE item (and its USD prim) may change after creation time (e.g.
     // parenting change caused by undo / redo of other commands in the undo
     // stack), so always return current data.
-    UsdPrim prim();
+    inline UsdPrim prim() { conditionalCreateItem(); return fItem->prim(); };
```

Comment:
@ppt-adsk  calling conditionalCreateItem routine before getting the prim from UsdSceneItem to avoid code duplication. 

The drawback is that we now have lost the constant here since we are now setting a member variable.

Reply:
Make fItem mutable and restore the const.  This is logical constness, which is fine.